### PR TITLE
Fix: Chart numbers stretch in runner when screen is resized

### PR DESCRIPTION
**Pull request by michael-adalo**
## Problem

Bar and Line charts appear squished or stretched when the screen width is updated.

## Solution

Version to publish: `0.2.22`

Ensure that when the screen is resized that the Bar and Line charts re-render to avoid the graphics appearing stretched.

https://user-images.githubusercontent.com/125075036/237060284-7ea54afe-f1e8-4dc3-a685-d4e4257a95b1.mp4

In addition to the fix, a little bit of housekeeping:

- `yarn.lock` is completely rewritten as it had become corrupted
- `@adalo/cli` updated to the latest version
- added GitHub PR Template
- updated Prettier and added config inline with `frontend`
- run Prettier against the codebase

## New Packages, Environment Variables (optional)

Removed
- `@storybook/addon-actions`
- `@storybook/addon-links `
- `@storybook/addons`
- `@storybook/react`

Updated
- `prettier` from `^1.19.1` to `2.5.1` (same used in `frontend`)
- `@adalo/cli` from `^0.0.42` to `^0.0.57` (latest)

### CODE REVIEW

**Comment by michael-adalo** [comment]
[Chart numbers stretch in runner when screen is resized](https://trello.com/c/xMrfAoP8/21-chart-numbers-stretch-in-runner-when-screen-is-resized)

File: adalo.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "author": "Kedar Bell",
```
Comment (danicunhac) [inline adalo.json:2]:
```suggestion
  "author": "Adalo",
```

**Review by danicunhac — APPROVED** [review]
We might need to change the author name in the component registry too in order to publish correctly, so please double check that

**Comment by michael-adalo** [comment]
> We might need to change the author name in the component registry too in order to publish correctly, so please double check that

thanks - the change is already published, but guess I could bump the version and attempt to publish again with the name change...

**Pull request merged**